draft-ietf-quic-transport.md

```diff
@@ -2271,8 +2269,8 @@ following layout:
 This design ensures that a stateless reset packet is - to the extent possible -
 indistinguishable from a regular packet with a short header.
 
-The message consists of a header byte, followed by an arbitrary number of random
-bytes, followed by a Stateless Reset Token.
+The message consists of the first two fixed bits of the packet header, followed
+by an arbitrary number of random bytes, followed by a Stateless Reset Token.
```

Comment:
Would it make sense to state like "followed by six (6) bits plus an arbitrary number of bytes in random"?

The fear I have with the current text is that it might imply that Stateless Reset Token starts at the 2 + 8n bit of the packet.